tests/testthat/helper-config.R

```diff
@@ -1,4 +1,5 @@
 ### lib
+library(golem)
```

Comment:
```suggestion

```

I don't think we need to re-load golem here?

Reply:
this is needed to use the run test button : 

![image](https://user-images.githubusercontent.com/10470699/97872434-5192e700-1d16-11eb-84de-fc1793415260.png)